### PR TITLE
Add make slice custom function.

### PR DESCRIPTION
**Pull request by yeweidaniel**

### CODE REVIEW

File: internal/template/funcmap.go [hunk 1]
```diff
@@ -139,3 +140,8 @@ func regexReplaceAll(regex string, s string, repl string) (string, error) {
 	}
 	return r.ReplaceAllString(s, repl), nil
 }
+
+// makeSlice combines the arguments into an array and returns the array.
+func makeSlice(args ...string) []string {
```
Comment (xingao267) [inline internal/template/funcmap.go:145]: we can probably simply name it `list` and extend it to be interface{}

```
func list(v ...interface{}) []interface{} {
	return v
}
```

Comment (MartinPetkov) [reply internal/template/funcmap.go:145]: +1

Comment (yeweidaniel) [reply internal/template/funcmap.go]: Done

**Comment by MartinPetkov** [comment]
The error from the presubmit job is this:
```
golangci/golangci-lint info installed /usr/local/bin/golangci-lint
The following files are not formatted:
internal/template/template_test.go
```

Try running `build/go_check.sh`. You may need to install [golangci-lint](https://github.com/golangci/golangci-lint).

**Comment by yeweidaniel** [comment]
> The error from the presubmit job is this:
> 
> ```
> golangci/golangci-lint info installed /usr/local/bin/golangci-lint
> The following files are not formatted:
> internal/template/template_test.go
> ```
> 
> Try running `build/go_check.sh`. You may need to install [golangci-lint](https://github.com/golangci/golangci-lint).

Running golangci-lint on the template_test.go file returns `template_test.go:242:10: undeclared name: makeSlice`, which seems to be related to an existing issue (https://github.com/golangci/golangci-lint/issues/1574).

**Comment by MartinPetkov** [comment]
Can you run it against all files instead?

**Comment by MartinPetkov** [comment]
Please run `gofmt .` to pass the CI check. This is the diff:
```
The following files are not formatted:
internal/template/template_test.go
To fix, run `gofmt .`
Diffs:
diff -u internal/template/template_test.go.orig internal/template/template_test.go
--- internal/template/template_test.go.orig	2021-03-15 16:31:23.760132099 +0000
+++ internal/template/template_test.go	2021-03-15 16:31:23.760132099 +0000
@@ -231,11 +231,11 @@
 	}{
 		{
 			inputs: []interface{}{"a", "b", "c"},
-			want: []interface{}{"a", "b", "c"},
+			want:   []interface{}{"a", "b", "c"},
 		},
 		{
 			inputs: []interface{}{1, 2, 3},
-			want: []interface{}{1, 2, 3},
+			want:   []interface{}{1, 2, 3},
 		},
 	}
 	for _, tc := range tests {
```

**Comment by yeweidaniel** [comment]
Fixed formatting, PTAL, thanks.

**Review by MartinPetkov — APPROVED** [review]

**Pull request merged**